crates/store/src/state.rs

```diff
@@ -280,15 +277,15 @@ impl State {
                     },
                 };
 
-                let merkle_path = note_tree.get_note_path(note_index);
+                let sparse_merkle_path = note_tree.open(note_index);
 
                 let note_record = NoteRecord {
                     block_num,
                     note_index,
                     note_id: note.id().into(),
                     metadata: *note.metadata(),
                     details,
-                    merkle_path,
+                    merkle_path: sparse_merkle_path.into(),
```

Comment:
Similar comment as above about conversions between sparse and regular Merkle paths.